modules/schedule_module/jsx/scheduleIndex.js

```diff
@@ -0,0 +1,576 @@
+        DCCID: row.DCCID,
+        PSCID: row.PSCID,
+        Session: sessionID,
+        AppointmentDate: row.Date,
+        AppointmentTime: row.Time,
+        AppointmentType: row['Appointment Type'],
+        SessionFieldOptions: sessionObj,
+   };
+   this.setState({formData: rowObj});
+ // ready setup edit form
+}
+  openModal() {
+    this.setState({showModal: true});
+  }
+  closeModal() {
+    this.setState({
+      formData: {},
+      showModal: false,
+      editModal: false,
+    });
+  }
+  /**
+   * Modify behaviour of specified column cells in the Data Table component
+   *
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {object} row - row content indexed by column
+   *
+   * @return {*} a formated table cell for a given column
+   */
+  formatColumn(column, cell, row) {
+    let result = <td>{cell}</td>;
+    switch (column) {
+    case 'PSCID':
+      let url = loris.BaseURL + '/' + row['DCCID'] + '/';
+      result = <td><a href ={url}>{cell}</a></td>;
+      break;
+    case 'Visit Label':
+      let visit = loris.BaseURL + '/instrument_list/?candID=' + row['DCCID'] + '&sessionID=' + row['Edit'];
+      result = <td><a href ={visit}>{cell}</a></td>;
+      break;
+    case 'Edit':
+      result = <td><button className="btn btn-default" onClick={() => this.edit(row)}><span className="glyphicon glyphicon-edit"></span> Edit</button></td>;
+      break;
+    case 'Delete':
+      result = <td><button className="btn btn-default" onClick={() => this.deleteConfirm(row['Delete'])}><span className="glyphicon glyphicon-trash"></span> Delete</button></td>;
+      break;
+    case 'Data Entry Status':
+      let css='label label-default';
+      if (cell==='Complete' ) {
+        css='label label-success';
+       }
+      if (cell==='No Data Found' ) {
+        css='label label-danger';
+       }
+      if (cell==='In Progress' || cell==='Not Started') {
+        css='label label-warning';
+       }
+      result = <td><span className={css}>{cell}</span></td>;
+      break;
+    case 'Appointment Type':
+      result = <td>{row['AppointmentTypeName']}</td>;
+      break;
+    }
+    return result;
+}
+  renderScheduleFormButton() {
+   if (this.state.editModal) {
+     return (<ButtonElement
+            name="edit"
+            label="Edit Appointment"
+            type="submit"
+            buttonClass="btn btn-sm btn-success"
+           />
+     );
+    } else {
+        return (<ButtonElement
+            name="create"
+            label="Create Appointment"
+            type="submit"
+            buttonClass="btn btn-sm btn-success"
+           />
+         );
+    }
+  }
+  renderScheduleForm() {
+	  console.log(this.state.formData);
+  let year = new Date();
+  let minYear = year.getFullYear();
+    const title = this.state.editModal ? 'Edit Appointment' : 'Add Appointment';
+    return (
+      <Modal
+        title= {title}
+        onClose={this.closeModal}
```

Comment:
Same here, console.log should be gone and the indentation is off in this function